test/client.nim

```diff
@@ -0,0 +1,66 @@
+import # nim libs
+  json, options, os, times, unittest
+
+import # vendor libs
+  chronos, confutils, json_serialization, sqlcipher, web3/conversions as web3_conversions
+
+import # nim-status libs
+  ../nim_status/[accounts, client, config, conversions, database, settings],
+  ./test_helpers
+
+procSuite "client":
+  asyncTest "client":
+
+    let dataDir = currentSourcePath.parentDir() / "build" / "data"
```

Comment:
Do we know if this will cause issues with Windows paths? @jrainville might have a suggestion. We handle similar situations in desktop like so: https://github.com/status-im/status-desktop/blob/22370e0b343da455378fd09cb322d2234241d6f8/src/app/utils/image_utils.nim#L3

Reply:
How does the `/` work? Is that a standard Nim way of writing paths? I'm asking because I would have assumed you'd have to write it like this: `currentSourcePath.parentDir() & "/build/data"`

So if it's some kind of Nim function, maybe it already replaces the `/` with the right one in the OS. If so, that's pretty cool!

Reply:
It comes from Nim's built-in `os` module, should work fine on Windows.